test/core/tsi/crl_ssl_transport_security_test.cc

```diff
@@ -390,6 +434,8 @@ std::string TestNameSuffix(
   return "TLS_1_3";
 }
 
+// TODO(gtcooke94) Add nullptr issuer test cases
```

Comment:
Do we need to address this TODO now?

Reply:
This is the hard-to-test thing we were discussing earlier

Reply:
Gotcha. Can you expand on the TODO and explain that we will come back to this when we factor out the callback into the util file?